source/appModules/soffice.py

```diff
@@ -433,6 +475,36 @@ def _backspaceScriptHelper(self, unit: str, gesture: inputCore.InputGesture):
 		else:
 			log.warning('Backspace did not remove text as expected.')
 
+	@script(
+		gestures=[
+			# bold
+			"kb:control+b",
+			# italic
+			"kb:control+i",
+			# underline
+			"kb:control+u",
+			# superscript
+			"kb:control+shift+p",
+			# subscript
+			"kb:control+shift+b",
+			# align left
+			"kb:control+l",
+			# align center
+			"kb:control+e",
+			# align right
+			"kb:control+r",
+			# justified
+			"kb:control+j",
+		]
+	)
+	def script_toggleTextAttribute(self, gesture):
+		# reset time and enable announcement of toggled toolbar buttons
+		# s. SymphonyButton#event_stateChange

```

Comment:
typo here

Reply:
Changed to this now:

    # see L{SymphonyButton.event_stateChange}

Is this the right way to reference that method? I've seen that being used elsewhere.

(From a quick search for Sphinx examples, I've found mentions of syntax using `:func:<method>`, which would potentially mean

    # see :func:`SymphonyButton.event_stateChange`

here? But I didn't see such syntax being used anywhere in the NVDA code base, so used what I saw elsewhere for now.)

Reply:
We have only just migrated to using Sphinx for new code instead of epydoc, I think using the Sphinx method is preferred here

Reply:
OK, switched to the Sphinx method (as I understand it) now.